content/management/v1/core-resources/collaborators/add-collaborator.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Add a Collaborator
+---
+
+You can set some of the fields available in the collaborator object, below we only list the properties in the example and the possible required fields.
+
+| Property | Description |
+|---|---|
+| `collaborator` | The [collaborator object](#core-resources/collaborators/the-collaborator-object) |
+| `email` | Email that will be used in the space interface for collaborator or collaborator SSO ID **required** |
+| `role` | Role name of the collaborator, could be admin, editor or custom roles, which is set to `id` and set to `multi` if you have more than one role **required** |
+| `space_id` | Numeric id of the collaborator space **required** |
+| `space_role_id` | Numeric id of the space role connected with collaborators, usually `null` with more than one collaborator  **required** |
```

Comment:
@iamfortune there's an error here. In my message I wrote "with more than one collaborator" but it doesn't makes sense. I meant to say "role", not "collaborator", sorry.